### PR TITLE
🐛 Fix aws dynamodb table init function, make all AWS init funcs use the mql cache

### PR DESCRIPTION
**Pull request by preslavgerchev**
Followup to #2194

### CODE REVIEW

File: providers/aws/resources/aws_dynamodb.go [hunk 1]
```diff
@@ -102,14 +102,17 @@ func initDynamoDbTable(runtime *plugin.Runtime, args map[string]*llx.RawData) (m
 	}
 	dynamodb := obj.(*mqlAwsDynamodb)
 
-	rawResources, err := dynamodb.tables()
+	rawResources := dynamodb.GetTables()
 	if err != nil {
 		return nil, nil, err
 	}
```
Comment (mariuskimmina) [inline providers/aws/resources/aws_dynamodb.go]: This is now checking an error that has been checked before and can be removed I think

File: providers/aws/resources/aws_dynamodb.go [hunk 2]
```diff
@@ -360,14 +363,17 @@ func initAwsDynamodbGlobaltable(runtime *plugin.Runtime, args map[string]*llx.Ra
...
 	dynamodb := obj.(*mqlAwsDynamodb)
 
-	rawResources, err := dynamodb.globalTables()
+	rawResources := dynamodb.GetGlobalTables()
 	if err != nil {
 		return nil, nil, err
 	}
```
Comment (mariuskimmina) [inline providers/aws/resources/aws_dynamodb.go]: This is also now checking an error that has been checked before and can be removed I think

**Review by mariuskimmina — COMMENTED** [review]
Nicely done, just some small suggestions

File: providers/aws/resources/aws_dynamodb.go (hunk 2 above)
Comment (preslavgerchev) [reply providers/aws/resources/aws_dynamodb.go]: great catch, resolved

**Review by mariuskimmina — APPROVED** [review]

**Pull request merged**